File: 0709. To Lower Case/solution.py
class Solution:
    def toLowerCase(self, s: str) -> str:
        s = list(s)
        for idx, letter in enumerate(s):
            if letter >= 'A' and letter < 'a':
                s[idx] = chr(ord(letter) + 32)
                
        return "".join(s)

File: 0709. To Lower Case/solution.cpp
class Solution {
public:
    string toLowerCase(string s) {
        stringstream result;
        for (auto ch: s) {
            char tmp = ch;
            if (ch >= 'A' && ch < 'a') {
                tmp += ' ';
            }
            result << tmp;
        }
        
        return result.str();
    }
};
